### ISSUE TITLE
Builds using 3.7.0-beta cannot be submitted to the App Store

### ISSUE THREAD

**Issue by eliperkins** [segmentio/analytics-ios#794]
Here's the error I just got from uploading a build of ours that uses 3.7.0-beta from Carthage.

```
[17:00:04]: ERROR ITMS-90060: "This bundle is invalid. The value for key CFBundleShortVersionString '3.7.0-beta' in the Info.plist file must be a period-separated list of at most three non-negative integers."
Return status of iTunes Transporter was 1: ERROR ITMS-90060: "This bundle is invalid. The value for key CFBundleShortVersionString '3.7.0-beta' in the Info.plist file must be a period-separated list of at most three non-negative integers."
The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.
```

This means that any Swift 4 project that uses this repo with Carthage cannot submit their application to the App Store.

The fix should be to remove the `-beta` suffix from the version in the Info.plist.

**Comment by ksm** [comment]
Just hit the same issue.

**Comment by f2prateek** [comment]
As a workaround - you could use one of the stable non-beta releases for now.

@fathyb could you take a look at this one?

**Comment by eliperkins** [comment]
There is no stable non-beta release for projects which use Swift 4.

**Comment by fathyb** [comment]
We've [released `3.7.0-beta.1`](https://github.com/segmentio/analytics-ios/blob/master/CHANGELOG.md#version-370-beta1-23rd-october-2018) with a fix, thanks to @eliperkins! Feel free to reopen if you're still experiencing this issue.
